src/angularjs/src/app/components/map/map.constants.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+  'use strict';
+
+  var config = {
+    baseLayers: {
+        'Stamen' : {
```

Comment:
It should be trivial, can we address #345 here as well by switching to positron?